src/Tasks/Common/Resources/Strings.resx

```diff
@@ -832,4 +832,8 @@ You may need to build the project on another operating system or architecture, o
     <value>NETSDK1181: Error getting pack version: Pack '{0}' was not present in workload manifests.</value>
     <comment>{StrBegin="NETSDK1181: "}</comment>
   </data>
+  <data name="Net6NotCompatibleWithDev16" xml:space="preserve">
+    <value>NETSDK1182: Using .NET 6.0 in Visual Studio 2019 is not a supported scenario.</value>
```

Comment:
Have we had PM review of this string?  I'd suggest the following:

```suggestion
    <value>NETSDK1182: Targeting .NET 6.0 in Visual Studio 2019 is not supported.</value>
```

@KathleenDollard

Reply:
@KathleenDollard suggested this string, but she mentioned that she might update it again. @KathleenDollard I'll let you handle changing the string in this PR if that's okay with you.

Reply:
It's not as simple as updating this in the PR, because you have to run a build to update all of the .xlf files (or manually update a bunch of them).